teuthology/task/ceph_ansible.py

```diff
@@ -411,6 +416,8 @@ def run_playbook(self):
             'mkdir',
             run.Raw('~/ceph-ansible'),
             run.Raw(';'),
+            run.Raw('cd ~'),
```

Comment:
Are you sure you need this? what is the issue? All test runs look fine?

Reply:
A run for me broke until I added this.

Reply:
How did it break?

Reply:
Okay.  maybe I don't need this.  I have run a couple of tests without it and not failed.